code/modules/clothing/spacesuits/rig/suits/light.dm

```diff
@@ -57,10 +54,9 @@
 	initial_modules = list(
 		/obj/item/rig_module/ai_container,
 		/obj/item/rig_module/power_sink,
-		/obj/item/rig_module/datajack,
 		/obj/item/rig_module/electrowarfare_suite,
-		/obj/item/rig_module/voice,
-		/obj/item/rig_module/vision,
+		/obj/item/rig_module/vision/meson,
```

Comment:
mesons are still NVG+ tho. Since you can see through walls (not people tho) + still gives nvg vision.

Reply:
i can change it if you want, was just thinking that meson is theoretically more common

Reply:
also meson doesn't let you see objects that you wouldn't be able to "see" without it, in the same room as you are, so it is worse in that manner

Reply:
well yeah, mesons are everywhere. Better having this than thermals. Also probably gives the rig some niche or reason to be used since it lacks all other "advantages".

Also fuck. I was supposed to do a normal comment, not an actual review. My bad about that.

Reply:
yeah i wanted to leave some sort of hud in currently since its helmet is literally a hud look-a-like but the default "vision" hud module is literally all of them besides sec/med and that's Slightly Overpowered, and meson hud is nice (basically infinite meson) while still being common enough to not eclipse station gear (mesons already exist on station they're just less convenient to use because of batteries)